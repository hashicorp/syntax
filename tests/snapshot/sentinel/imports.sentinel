# https://developer.hashicorp.com/sentinel/docs/language/imports

import "calendar"

import "name"

import "name1" as one
import "name2" as two
