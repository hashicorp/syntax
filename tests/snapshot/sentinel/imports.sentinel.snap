># https://developer.hashicorp.com/sentinel/docs/language/imports
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.sentinel comment.line.number-sign.sentinel
>
>import "calendar"
#^^^^^^ source.sentinel keyword.control.declaration.sentinel
#      ^ source.sentinel
#       ^ source.sentinel string.quoted.double.untitled
#        ^^^^^^^^ source.sentinel string.quoted.double.untitled
#                ^ source.sentinel string.quoted.double.untitled
>
>import "name"
#^^^^^^ source.sentinel keyword.control.declaration.sentinel
#      ^ source.sentinel
#       ^ source.sentinel string.quoted.double.untitled
#        ^^^^ source.sentinel string.quoted.double.untitled
#            ^ source.sentinel string.quoted.double.untitled
>
>import "name1" as one
#^^^^^^ source.sentinel keyword.control.declaration.sentinel
#      ^ source.sentinel
#       ^ source.sentinel string.quoted.double.untitled
#        ^^^^^ source.sentinel string.quoted.double.untitled
#             ^ source.sentinel string.quoted.double.untitled
#              ^ source.sentinel
#               ^^ source.sentinel keyword.other.sentinel
#                 ^^^^^ source.sentinel
>import "name2" as two
#^^^^^^ source.sentinel keyword.control.declaration.sentinel
#      ^ source.sentinel
#       ^ source.sentinel string.quoted.double.untitled
#        ^^^^^ source.sentinel string.quoted.double.untitled
#             ^ source.sentinel string.quoted.double.untitled
#              ^ source.sentinel
#               ^^ source.sentinel keyword.other.sentinel
#                 ^^^^^ source.sentinel
>